runtime/signals_nat.c

```diff
@@ -286,6 +286,29 @@ void caml_init_signals(void)
 #endif
 }
 
+/* Termination of signal stuff */
+
+#if defined(HAS_STACK_OVERFLOW_DETECTION)
```

Comment:
What is the reason for not handling the power and s390x case?

Reply:
Only being very minimal in what was back-ported

Reply:
I would tend to argue that keeping the distance between the 4.13 and 4.14 patches minimal (to share both implementations and potential bugs) is as useful as minimizing the patch size itself.

Reply:
I pushed that too - my paranoia is that technically-speaking resetting those signals is not part of the bug-fix (the bug was freeing the alternate signal stack and before doing that it is necessary to remove the handler)